kubernetes-deploy.gemspec

```diff
@@ -22,9 +22,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
   spec.add_dependency "activesupport", ">= 4.2"
   spec.add_dependency "kubeclient", "~> 2.3"
+  spec.add_dependency "ejson", "1.0.1"
```

Comment:
It turns out that the gem includes the Go binary for linux and darwin:

```
$ wget https://rubygems.org/downloads/ejson-1.0.2.gem
$ gem unpack /Users/kir/Downloads/ejson-1.0.2.gem
Unpacked gem: '/Users/kir/Projects/opensource/kubernetes-deploy/ejson-1.0.2'
$ ls -la ejson-1.0.2/build
drwxr-xr-x  3 kir  staff  102 Apr 21 15:04 darwin-amd64
drwxr-xr-x  3 kir  staff  102 Apr 21 15:04 linux-amd64
```

So ignore my point.